src/components/preview-panel/latex-containier/light-wrapper.tsx

```diff
@@ -1,13 +1,27 @@
-import React, { FC } from "react";
-import styled from "@emotion/styled";
+import React, { FC, useEffect } from "react";
 import { SourceCodeWrapper } from "./latex-container.style";
 
-const Wrapper = styled(SourceCodeWrapper)``;
-
 type LightWrapperProps = {
   children: React.ReactNode;
 };
 
-export const LightWrapper: FC<LightWrapperProps> = ({ children }) => (
-  <Wrapper>{children}</Wrapper>
-);
+export const LightWrapper: FC<LightWrapperProps> = ({ children }) => {
+  useEffect(() => {
+    const link = document.createElement("link");
+    link.rel = "stylesheet";
+    link.href =
+      "https://cdn.jsdelivr.net/npm/prismjs@1.26.0/themes/prism.min.css";
+
+    document.head.appendChild(link);
+
+    return () => {
+      document.head.removeChild(link);
+    };
+  }, []);
+
+  return (
+    <div>
+      <SourceCodeWrapper>{children}</SourceCodeWrapper>
+    </div>
```

Comment:
why not set `height` to `100%` (better for **less** code display), it will include the scrollbar and cover copy icon (bad in **more** code display).

bad user experience.

maybe will fix it in the future.